docs/security/trust/trust_key_mng.md

```diff
@@ -64,6 +64,16 @@ Before backing them up, you should `tar` them into an archive:
 $ umask 077; tar -zcvf private_keys_backup.tar.gz ~/.docker/trust/private; umask 022
 ```
 
+## Hardware storage and signing
+
+Docker Content Trust can store and sign with root keys from a Yubikey 4.
+The Yubikey will be prioritized over keys stored in the filesystem; if a root key
+is not found locally when initializing a new repository with content trust, Docker
+Engine will prioritize creating a root key in a Yubikey if it can find one.
+Please consult the [Notary documentation](https://docs.docker.com/notary/advanced_usage/#use-a-yubikey) for more details.
```

Comment:
thanks, done!  I've also updated other notary documentation links to be consistent